lib/repositories/movie/Impl/movie_from_json.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+
+import '../i_movie_repository.dart';
+import '../../../data_classes/movie.dart';
+
+class MoviesFromJson extends MovieRepository {
+  static const moviesPath = 'jsonData/movies.json';
+  static late List<Movie> parsedMovies;
+
+  static Future<List<Movie>> parseMovies() async {
+    final jsonPath = await rootBundle.loadString('jsonData/movies.json');
+    return List.from(jsonDecode(jsonPath).map((e) => Movie.fromJson(e)));
+  }
+
+  @override
+  Future<List<Movie>> getMovies() async {
+    if (parsedMovies.isEmpty) {
+      parsedMovies = await parseMovies();
+    }
+    return parsedMovies;
+  }
+
+  MoviesFromJson() {
+    parsedMovies = [];
+  }
```

Comment:
Why do you need to do this?

Reply:
Because if I don´t do that, I get the following error:
'Late initialization error: Field parsedMovies has not been initialized'

Reply:
Deleted the constructor and made parsedMovies a late field. Solved